docs/dashboards/highcharts-component.md

```diff
@@ -177,6 +177,27 @@ columnAssignment: [{
 ```
 <iframe style="width: 100%; height: 600px; border: none;" src=https://www.highcharts.com/samples/embed/dashboards/components/highcharts-column-assignment-keys-data allow="fullscreen"></iframe>
 
+### Multiple connectors
+
+Highcharts Component supports also more than one data source. This can be achieved by setting the `connector` option to an array of objects rather than a single object.
```

Comment:
Suggested rewrite: The Highcharts component also supports ...

Reply:
I would also rephrase the second sentence. I suggest:
 `That means the connector option should be configured as an array of objects...`